src/proto/ctl/storage_nvme.proto

```diff
@@ -41,9 +41,11 @@ message NvmeControllerResult {
 }
 
 message ScanNvmeReq {
-	bool Health = 1; // Retrieve NVMe device health statistics
-	bool Meta = 2; // Retrieve metadata relating to NVMe device
-	bool Basic = 3; // Strip NVMe device details to only basic
+	bool Health = 1;	// Retrieve NVMe device health statistics
+	bool Meta = 2;		// Retrieve metadata relating to NVMe device
+	bool Basic = 3;		// Strip NVMe device details to only basic
+	uint64 MetaSize = 4;	// Size of the metadata blob
```

Comment:
It looks like these fields are being added pre-emptively, and are not currently being used. I would prefer to add them when needed.

Reply:
They are used in the `updateInUseBdevs()` function to call `GetBioHealth()` function with a ScanNvmeReq containing the size of the RDB and the metadata size.

Reply:
ah, yes of course, apologies